plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -336,6 +338,16 @@ object StmtConversionVisitor : FirVisitor<Exp, StmtConversionContext<ResultTrack
         data.signature.receiver?.toLocalVar()
             ?: throw IllegalArgumentException("Can't resolve the 'this' receiver since the function does not have one.")
 
+    override fun visitTypeOperatorCall(typeOperatorCall: FirTypeOperatorCall, data: StmtConversionContext<ResultTrackingContext>): Exp {
+        val argument = typeOperatorCall.arguments.elementAtOrNull(0)?.accept(this, data)
```

Comment:
Since you're going to assume this is non-null later, it seems like it would be easier to just use `arguments[0]` here (or `!!`).